src/components/ColorPicker/colorPicker.test.tsx

```diff
@@ -0,0 +1,34 @@
+import { ColorPaletteCustom } from "../../colors";
+import {
+  getColorNameAndShadeFromColor,
+  colorPickerHotkeyBindings,
+} from "./colorPickerUtils";
+import { render, screen } from "@testing-library/react";
+
+test("getColorNameAndShadeFromColor returns null when no matching color is found", () => {
```

Comment:
lets wrap it inside `describe` and use `it` block to maintain consistency. Also rename the file to `colorPickerUtils.ts` and lets add tests for all utils here.
You can follow the same structure we have in https://github.com/excalidraw/excalidraw/blob/master/src/tests/packages/utils.test.ts (A describe block for each util with tests for each util as needed for different scenarios

Reply:
> colorPickerUtils.ts

but how can we write test inside it, I've renamed it to ### colorPicker.test.ts

Reply:
Oops! @Thedineshk24  I meant rename the file to `colorPickerUtils.test.ts` since it contains the tests related to the utils only

Reply:
@colorPickerUtils oh okay got it.

Reply:
@ad1992 kindly check now renamed file to colorPickerUtils.test.ts